src/systemctl/systemctl.c

```diff
@@ -249,6 +249,7 @@ static int systemctl_help(void) {
                "  soft-reboot                         Shut down and reboot userspace\n"
                "  exit [EXIT_CODE]                    Request user instance or container exit\n"
                "  switch-root [ROOT [INIT]]           Change to a different root file system\n"
+               "  sleep                               Sleep the system\n"
```

Comment:
"Put the system to sleep"

Reply:
> "Put the system to sleep"

That sounds like a *nice* way of euthanizing the system :))